src/sentry/receivers/releases.py

```diff
@@ -71,6 +71,7 @@ def resolved_in_commit(instance, created, **kwargs):
                                 'commit': instance.id,
                             }
                         )
+                    GroupAssignee.objects.assign(group=group, assigned_to=user_list[0])
```

Comment:
I'm not going to block on this, but in what case do we have multiple users here? We're creating multiple activity items, but we're blindly picking the first user here.

I think it's fine, I'm just curious in what case we may hit more than 1. And in that case, why are we creating multiple activity entries to go with it? Maybe it's just better to pick the first one and only create the activity item for that one user too?